src/main/java/ru/yandex/practicum/filmorate/storage/InMemoryFilmStorage.java

```diff
@@ -0,0 +1,54 @@
+package ru.yandex.practicum.filmorate.storage;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.server.ResponseStatusException;
+import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.utils.ValidationException;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+@Component
+public class InMemoryFilmStorage implements FilmStorage {
+    private final HashMap<Integer, Film> films = new HashMap<>();
```

Comment:
Определяй интерфейс, а не конкретную реализацию
```suggestion
    private final Map<Integer, Film> films = new HashMap<>();
```